core/src/main/java/org/frankframework/jdbc/StoredProcedureQuerySender.java

```diff
@@ -278,6 +280,36 @@ private Message getResult(CallableStatement callableStatement, boolean alsoGetRe
 		return Message.asMessage(result);
 	}
 
+	static SQLType mapParameterTypeToSqlType(IDbmsSupport dbmsSupport, Parameter.ParameterType parameterType) {
```

Comment:
Dit is misschien juist wel handig in een Util class

Reply:
Teruggezet.